.idea/runConfigurations/Check.xml

```diff
@@ -7,7 +7,9 @@
     <option name="emulateTerminal" value="false" />
     <option name="backtrace" value="SHORT" />
     <option name="workingDirectory" value="file://$PROJECT_DIR$" />
-    <envs />
+    <envs>
+      <env name="SPATIAL_LIB_DIR" value="./dependencies" />
```

Comment:
?

Reply:
I missed setting this environment variable in #142